File: MPI/mpicheck/tests/functions/1.3/MPI_Get_version.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int *var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Get_version(var_0, var_1);
    ret = PMPI_Get_version(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.3/MPI_Get_version.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_get_version(var_0, var_1, var_2)
       call pmpi_get_version(var_0, var_1, var_2)
        end program main
    